### ISSUE TITLE
Spanish translation / Traducción al español

### ISSUE THREAD

**Issue by awvwgk** [awvwgk/fpm-docs#4]
This thread tracks the translation of the documentation to Spanish. Feel free to post either in Spanish or English in this threads, whatever seems most appropriate.

cc @aslozada

**Comment by aslozada** [comment]
The title of the index.html without translation

index.po (Spanish)
![espanol_po](https://user-images.githubusercontent.com/3170541/143629536-b7110a8e-2a50-42e6-9d86-1a89187613e8.png)
![Espanol](https://user-images.githubusercontent.com/3170541/143629968-05b872d5-bb75-48bf-a654-bbf15fb9238c.png)

index.po (German)

![deutsch_po](https://user-images.githubusercontent.com/3170541/143630953-768632e1-9976-4ea8-bd67-8acb2c084f75.png)
![deutsch](https://user-images.githubusercontent.com/3170541/143631429-52f854e4-decb-44c8-8bdb-d7bb4fbd8465.png)

**Comment by awvwgk** [comment]
I think this might be a bug in sphinx-design, the `Fortran package manager` is a card title which apparently not captured with babel, similar issue for the button-ref, I will report those upstream.

**Comment by aslozada** [comment]
Some warnings when translating menu options
```

fpm-docs/pages/index.md:1: WARNING: inconsistent term references in translated message. original: ['tutorial'], translated: []
fpm-docs/pages/index.md:1: WARNING: inconsistent term references in translated message. original: ['how-to'], translated: []
fpm-docs/pages/index.md:1: WARNING: inconsistent term references in translated message. original: ['design'], translated: []
fpm-docs/pages/index.md:1: WARNING: inconsistent term references in translated message. original: ['spec'], translated: []

```
Result

![menu](https://user-images.githubusercontent.com/3170541/143720324-8e06653a-cc99-461d-902a-4443dec8e10f.png)

**Comment by awvwgk** [comment]
Yeah, I found this one as well. Most likely due to https://github.com/executablebooks/sphinx-design/issues/44, another issue with sphinx-design, resulting from the translation of the button-ref labels. I just went with not translating those for now until we get a fix upstream.

**Comment by aslozada** [comment]
Description in "How-to" section:

file ``` how-to.po. installation.md:3```

---
**Installing fpm**
This how-to guide covers the installation of  Fortran on various platforms.

---

**Fortran** or **fpm**?

**Comment by aslozada** [comment]
Buttons lose features after translation. (Text-color, Background-color, action)
is this another issue with sphinx-design?

![buttons](https://user-images.githubusercontent.com/3170541/144159595-149fc770-9973-4303-a080-daaf2b7390c0.png)

**Comment by awvwgk** [comment]
I changed the markup to capture the card titles, I'll look into a way to get the button-ref directives working. I could make the whole card clickable as alternative

**Comment by aslozada** [comment]
Hi Sebastian @awvwgk,
The languages links are missing in [fpm.fortran-lang.org](https://fpm.fortran-lang.org/)
![missing](https://user-images.githubusercontent.com/3170541/164567318-6ee193d3-f2ba-4ea2-88d9-c4b11d58e45c.png)

**Comment by awvwgk** [comment]
An update of the sphinx-book-theme broke your side. I'll pin it to 0.2.0 and than figure out later what was changed.
